### PR TITLE
Updated the editor docs to include the Revisions feature

### PR DESCRIPTION
**Pull request by jadesola123**
I updated the "editing existing pages" in the editor's manual to include the revisions feature.

I also included gifs and screenshots from gitpod to make the documentation easier to understand for editors.

Solving  [issue](https://github.com/wagtail/guide/issues/176) opened by @yhoiseth

### CODE REVIEW

**Comment by lb-** [comment]
@jadesola123 Thanks for this but I'm not sure we are accepting new contributions to the user guide section. 

The user guide will now be an external Wagtail site, where the content is managed within a Wagtail instance itself. 

I'll let @thibaudcolas have a look at this and see if it's worth merging/ reviewing as we may end up deleting these pages very soon.

**Comment by lb-** [comment]
See issue that's now moved to Wagtail guide site project. 
https://github.com/wagtail/guide/issues/176

**Comment by jadesola123** [comment]
Thank you for the feedback @lb- , looking forward to your feedback @thibaudcolas

File: docs/editor_manual/editing_existing_pages.md [hunk 1]
```diff
@@ -17,6 +17,35 @@ Here is how you can access the edit screen of an existing page:
...
+
+Once you're on the history page, you'll see:
+
+- Actions that have been done on that particular page.
+- The user that carried out that action.
+- The date/time the action occurred.
+
+You can also apply a filter to find specific versions of your content.
+
+![The actions and filters for page history](../_static/images/page_history_filters.gif)
```
Comment (thibaudcolas) [inline docs/editor_manual/editing_existing_pages.md:36]: No need for you to redo this – but I’d prefer we avoid the GIF. They’re very hard for us to keep up to date. I’ll switch over to a PNG when merging.

**Review by thibaudcolas — COMMENTED** [review]
This is looking solid @jadesola123. We’ll definitely be able to make use of this. I might merge this and then delete the content shortly after, but I’ll make sure we add this to the separate site as well.

I’ll review in more detail tomorrow.

**Comment by jadesola123** [comment]
Thank you very much for the feedback! @thibaudcolas

**Comment by thibaudcolas** [comment]
@jadesola123 I can confirm I’ll be able to get this into the site. One thing I wanted to check with you is – in your pull request you mention:

```
Solving issue #5711 opened by @yhoiseth 
```

This issue isn’t related to that pull request, wasn’t opened by that user, and I don’t see anything about revisions from them. Could you double check which issue this is for?

**Pull request closed without merging**